assets/model_monitoring/components/tests/e2e/test_data_joiner_e2e.py

```diff
@@ -105,3 +106,20 @@ def test_data_joiner_empty_result_failed(
         )
 
         assert pipeline_job.status == 'Failed'
+
+    def test_data_joiner_common_columns_failed(
```

Comment:
a minor thought:  would it be easier to test by UT? because adding more and more e2e tests may make the gated tests run very long.